versions.mk

```diff
@@ -1,16 +1,16 @@
 # https://github.com/helm/chart-testing/releases
-CHART_TESTING_VERSION := 3.12.0
+CHART_TESTING_VERSION := 3.14.0
 # https://github.com/kubernetes-sigs/controller-tools/releases
-CONTROLLER_TOOLS_VERSION := v0.18.0
+CONTROLLER_TOOLS_VERSION := v0.19.0
 # https://github.com/helm/helm/releases
-HELM_VERSION := 3.18.1
+HELM_VERSION := 3.19.4
```

Comment:
I don't think we have to stick to v3.x. Let's use v4.0.4, which is the latest version.